weaveexec/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"flag"
+	. "github.com/weaveworks/weave/common"
+	"github.com/weaveworks/weave/proxy"
+	"net/http"
+)
+
+func main() {
+	var target, listen string
+	var debug bool
+
+	flag.StringVar(&target, "H", "unix:///var/run/docker.sock", "docker daemon URL to proxy")
+	flag.StringVar(&listen, "L", ":12375", "address on which to listen")
+	flag.BoolVar(&debug, "debug", false, "log debugging information")
+	flag.Parse()
+
+	if debug {
+		InitDefaultLogging(true)
+	}
+
+	p, err := proxy.NewProxy(target)
```

Comment:
You're not checking the value of `err`, and the compiler isn't noticing the unused value because of the assignment on line 32. I wonder if it's a best practice to always use `:=` for error values due to this?